modules/kubernetes/ingress-nginx/templates/values.yaml.tpl

```diff
@@ -65,6 +70,15 @@ controller:
     %{~ endif ~}
   %{~ endif ~}
 
+  %{~ if datadog_enabled ~}
```

Comment:
Why should this only be applied for datadog? Don't look like it would hurt the general config.

Reply:
It's only being used by datadog as of now, which means it shouldn't be added to environments where it's not being used.